jujugui/static/gui/src/app/components/profile/charm-list/charm-list.js

```diff
@@ -87,11 +86,11 @@ class ProfileCharmList extends React.Component {
   /**
     Display a tag in the store.
     @param tag {String} The name of the tag.
-    @param evt {Object} The click event.
+    @param e {Object} The click event.
   */
-  _handleTagClick(tag, evt) {
```

Comment:
I don't care either way, I'd just like consistency.

Reply:
Ahh it might have been before your time, I'd always write `e` then other people would come in future branches and change them to `evt`, now you're changing them back haha